developmentalparsers/w2dynamics/parser.py

```diff
@@ -37,6 +37,7 @@ class ParameterParser(TextParser):
     def init_quantities(self):
         section_quantities = [
             Quantity(
+                #'parameter', f'{re_n} *((\w|-)+ *= *.+)',
```

Comment:
If you want to add special characters in addition to alphanumeric \w you can do [\w\-\|]+